### PR TITLE
:tada: The arrows allow the user to navigate from block to block

### LINKED ISSUE #64: Block navigation
We should be able to navigate from block to block using only the keyboard arrows

### PR DESCRIPTION
**Pull request by vanyle**
This resolves #64

### CODE REVIEW

File: opencodeblocks/graphics/view.py [hunk 1]
```diff
@@ -134,6 +134,85 @@ def middleMouseButtonRelease(self, event: QMouseEvent):
...
+        items = self.scene().items()
+        code_blocks = [i for i in items if isinstance(i, OCBBlock)]
+
+        # Pick the block with the center distance (x,y) such that:
+        # ||(x,y)|| is minimal but not too close to 0, where ||.|| is the infinity norm
+        # This norm was choosen because the movements it generates feel natural.
+        # x or y has the correct sign (depends on the key pressed)
+
+        dist_array = []
+        for block in code_blocks:
```
Comment (MathisFederico) [inline opencodeblocks/graphics/view.py:174]: In the future, we might want to only look at close blocks

File: opencodeblocks/graphics/view.py [hunk 2]
```diff
@@ -134,6 +134,85 @@ def middleMouseButtonRelease(self, event: QMouseEvent):
...
+        """
+        # The focusItem has priority for this event
+        if self.scene().focusItem() is not None:
+            return False
+        if len(self.scene().selectedItems()) > 0:
+            return False
+
+        key_id = event.key()
+        items = self.scene().items()
+        code_blocks = [i for i in items if isinstance(i, OCBBlock)]
```
Comment (MathisFederico) [inline opencodeblocks/graphics/view.py:166]: blocks in a more general name, as text blocks and so on might be in there

**Review by MathisFederico — APPROVED** [review]
Well done !

File: opencodeblocks/graphics/view.py (hunk 2 above)
Comment (vanyle) [reply opencodeblocks/graphics/view.py:166]: I will edit this in the pull request that resolves #28

**Pull request merged**